addons/account/models/res_config_settings.py

```diff
@@ -133,6 +133,10 @@ class ResConfigSettings(models.TransientModel):
                                      readonly=False)
     terms_type = fields.Selection(
         related='company_id.terms_type', readonly=False)
+    invoice_amount_total_words = fields.Boolean(
+        string="Total amount of invoice in letters",
+        related='company_id.display_invoice_amount_total_words',
+        readonly=False)
```

Comment:
```suggestion
        readonly=False
    )
```

Reply:
Changes from both feedbacks added. Thanks for the review. should I put the task in blocked as long as PR (https://github.com/odoo/odoo/pull/107084 has not been merged ?

Reply:
yep